src/move_basic.cpp

```diff
@@ -188,6 +202,8 @@ MoveBasic::MoveBasic(): tfBuffer(ros::Duration(30.0)),
 
     obstacle_dist_pub =
         ros::Publisher(nh.advertise<std_msgs::Float32>("/obstacle_distance", 1));
+    errorPub =
```

Comment:
Does Vector3 make sense here?

Reply:
Vector3 makes sense for the error, since there is no Vector2.  However, adding the commanded rotation in there is bogus, and that information is available from /cmd_vel, so I'll remove it.